### ISSUE TITLE
Issue on AWS Lambda

### ISSUE THREAD

**Issue by Rajivhost** [w3cj/stoker#1]
Hi @w3cj,

I have below error on using the lib on my aws lambda function code.

Seems to complain about the missing libs, but the lib is well install.

![image](https://github.com/user-attachments/assets/a92218e3-bf72-4881-bc76-948156cac407)

I also have some type errors

![image](https://github.com/user-attachments/assets/f8a5fe89-7e20-4c09-89b1-05a13b4b5ea1)

**Comment by w3cj** [comment]
For the first issue, you might need to install the peer dependency:

`@asteasolutions/zod-to-openapi`

Not totally sure about the second issue, but you can try importing the middlewares directly:

```ts
import notFound from "stoker/middlewares/not-found";
import onError from "stoker/middlewares/on-error";

```

**Comment by Rajivhost** [comment]
Hi @w3cj,

Installing the peer dependency as suggested not fixing the issue as we can see below.
It's possible that your lib automatically deal with this one to not be obligle to do that by ourself?

![image](https://github.com/user-attachments/assets/107f7a2e-78e7-4c3c-a987-b8cb59ef48c2)

![image](https://github.com/user-attachments/assets/33ac4657-fa3c-4f3d-a262-e943fa90343f)

**Comment by Rajivhost** [comment]
Also, I have strange imports behavior as we can see below.

I have errors doing this:

![image](https://github.com/user-attachments/assets/0087757a-2bb6-44f4-b966-ec2bcc10c9fc)

And seems works doing this:

![image](https://github.com/user-attachments/assets/9215ae67-9a98-4b9c-b8fd-ff6ae4b416af)

What is the problem here, any idea please?

**Comment by w3cj** [comment]
I published a new version of the library `1.2.3` with an updated build / type exports. Please try it and let me know if it resolves your issues.

The peer dependencies are not included directly because they are only required for certain helpers / utilities under `stoker/openapi`. Users of this library that do not use any of those helpers do not need to install this peer dependency.

**Comment by Rajivhost** [comment]
Here what I have now

![image](https://github.com/user-attachments/assets/77258782-6210-4018-aec6-515b26906947)

![image](https://github.com/user-attachments/assets/9e251800-b55c-4d4a-97d2-0126f555e1be)

**Comment by w3cj** [comment]
I published one more build update in version `1.2.4` if this does not work for you, you will need to set the `moduleResolution` option in your tsconfig to `bundler`

**Comment by Rajivhost** [comment]
Works for me now @w3cj 

Thanks a lot for your support!!
